recipes-ni/ni-utils/ni-utils_1.0.bb

```diff
@@ -5,15 +5,16 @@ LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda
 SECTION = "base"
 
 SRC_URI = "file://status_led \
-	   file://nisetbootmode.functions \
-	   file://nisetbootmode \
+           file://nisetbootmode.functions \
+           file://nisetbootmode \
+           file://nisetled \
```

Comment:
Can you explain why you are moving nisetled to this package?

Reply:
nisetled is the init script for status_led which is provided by this package - so moved it here.

Reply:
I liked the initscripts only package separation from before. The status_led is called by other scripts too directly, so it makes sense that is installed by ni-utils. Can you make initscripts depend on ni-utils instead?

Reply:
I get your point, but in my view there is a fundamental difference between initscript and other scripts calling status_led. If this initscript were to do other things in addition to calling status_led, it'd make sense to put it in initscripts, but the sole purpose of this initscript is to make sure status_led runs at boot time, so I feel like it should go in same package. We always ship initscripts associated with other software in the same package even when they're called outside the initscript, wouldn't we be breaking this convention if we split them in this case?

Reply:
> We always ship initscripts associated with other software in the same package even when they're called outside the initscript..

I didn't realize this was the case. I'm good with this then.